### PR TITLE
GraphQL-672: Change customer password

### PR DESCRIPTION
**Pull request by rafaelstz**
https://github.com/magento/graphql-ce/issues/672

Test cases:

```
\Magento\GraphQl\Customer\ChangeCustomerPasswordTest
- testChangePasswordIfCurrentPasswordIsEmpty
- testChangePasswordIfNewPasswordIsEmpty
- testChangePasswordIfCustomerIsLocked (see \Magento\GraphQl\Customer\UpdateCustomerTest::lockCustomer)
- tesChangeCustomerAddressIfAccountIsNotConfirmed
```

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/graphql-ce?pullRequest=705) <br/>All committers have signed the CLA.

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-5207](https://jira.corp.magento.com/browse/ENGCOM-5207) has been created to process this Pull Request

**Comment by lenaorobei** [comment]
@magento run all tests

**Review by lenaorobei — CHANGES_REQUESTED** [review]
@rafaelstz could you please address WebAPI Tests build issue. Thanks.
```
Failed asserting that exception message 'GraphQL response contains errors: Specify the "newPassword" value.
' contains 'Account is not confirmed.'.
```

File: dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/ChangeCustomerPasswordTest.php [hunk 1]
```diff
@@ -47,19 +63,19 @@ protected function setUp()
...
+        $newPassword = 'anotherPassword1';
 
-        $query = $this->getChangePassQuery($oldCustomerPassword, $newCustomerPassword);
-        $headerMap = $this->getCustomerAuthHeaders($customerEmail, $oldCustomerPassword);
+        $query = $this->getQuery($currentPassword, $newPassword);
+        $headerMap = $this->getCustomerAuthHeaders($customerEmail, $currentPassword);
 
         $response = $this->graphQlMutation($query, [], '', $headerMap);
-        $this->assertEquals($customerEmail, $response['changeCustomerPassword']['email']);
+        $this->assertEquals($customerEmail, $response['changePassword']['email']);
```
Comment (lenaorobei) [inline dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/ChangeCustomerPasswordTest.php]: Looks like should be `changeCustomerPassword`.

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-5207](https://jira.corp.magento.com/browse/ENGCOM-5207) has been created to process this Pull Request

**Pull request merged**